protocol/types.go

```diff
@@ -187,3 +189,34 @@ type VersionResponse struct {
 	// Build contains the timestamp at the time of the build.
 	Build time.Time `json:"build"`
 }
+
+// SupportedLanguagesRequest is a request to get the supported languages
+//proteus:generate
+type SupportedLanguagesRequest struct{}
+
+// SupportedLanguagesResponse is the reply to SupportedLanguagesRequest
+//proteus:generate
+type SupportedLanguagesResponse struct {
+	Response
+	// Drivers are the loaded drivers
+	Drivers []DriverDetails `json:"drivers"`
```

Comment:
I just realized that to be consistent with `SupportedLanguages` name, this should renamed as `Languages`, shouldn't it?
I'll rename it once we have a consensus about https://github.com/bblfsh/sdk/pull/259#discussion_r192006369 with @smola and @mcuadros.